docs/authentication/configuration/sign-up-sign-in-options.mdx

```diff
@@ -135,35 +139,35 @@ Clerk offers three verification methods:
 
 {/* TODO: Is this too much to add? */}
 
-Users are also able to link multiple social providers to their account depending on how you have set up your application. You can set up your application to use the [Account Portal User Profile page](/docs/customization/account-portal//user-profile-org-profile#user-profile), the prebuilt [`<UserProfile />`](/docs/components/user/user-profile) component, or you can [build your own custom user interface using the Clerk API](/docs/custom-flows/oauth-connections).
+Users can link multiple social providers to their account, depending on your application's setup. You can configure your application to use the [Account Portal User Profile page](/docs/customization/account-portal//user-profile-org-profile#user-profile), the prebuilt [`<UserProfile />`](/docs/components/user/user-profile) component, or [build your own custom user interface using the Clerk API](/docs/custom-flows/oauth-connections).
 
 To enable social connections:
 
 1. Navigate to the [Clerk Dashboard](https://dashboard.clerk.com/last-active?path=user-authentication/social-connections) and select your application.
 1. In the navigation sidebar, select **User & Authentication > Social connections**.
-1. For development instances, you can simply toggle on the social providers you would like to enable. For production instances, you will need to configure credentials for each social provider. Don't worry, there are [detailed guides](/docs/authentication/social-connections/overview) for each social provider to help you through the process.
+1. For development instances, you can simply toggle on the social providers you would like to enable. For production instances, you'll need to configure credentials for each social provider. Don't worry, there are [detailed guides](/docs/authentication/social-connections/overview) for each social provider to help you through the process.
 
 ## Web3 authentication
 
-Clerk provides [Web3 authentication](/docs/users/web3) as an option with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication factors, like passwords or email links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
+Clerk provides [Web3 authentication](/docs/users/web3)with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication factors, like passwords or email links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
```

Comment:
```suggestion
Clerk provides [Web3 authentication](/docs/users/web3) with MetaMask. As part of validating the accuracy of the returned Web3 account address, Clerk handles the signing of a transaction and verifying the signature. Because sign-in with MetaMask uses the same abstraction as our other authentication factors, like passwords or email links, other Clerk features like multi-factor authentication and profile enrichment work for MetaMask users out-of-the-box.
```